actix_full/src/main.rs

```diff
@@ -121,6 +121,17 @@ impl rwcst::AppImpl for App {
             })
         });
 
+        fn serve(&mut self) -> Result<()> {
+            let addr = SocketAddr::from(([127, 0, 0, 1], 3000));
+
+            let make_svc =
+                make_service_fn(|_conn| async { Ok::<_, Infallible>(service_fn(handle)) });
+
+            Server::bind(&addr).serve(make_svc);
+
+            Ok(())
+        }
```

Comment:
This should be made in another commit as this is not related.

Reply:
Done!